pocs/scheduler/dispatch.py

```diff
@@ -30,6 +30,8 @@ def get_observation(self, time=None, show_all=False, reread_fields_file=False):
                 defaults to time called
             show_all (bool, optional): Return all valid observations along with
                 merit value, defaults to False to only get top value
+            reread_fields_file (bool, optional): If the fields file should be reread
```

Comment:
And the reason for not making the change here in the params list (i.e. default =True).